# Langmuir Probe Experiment
## Prelab 1
### Characterizing the positive column of a glow discharge plasma

We want to explore the following key idea:

* Plasmas are ionized gases characterized by macroscopic parameters that include temperatures, densities, and pressure. The parameters specify  statistical distributions of the positions and velocities of the particles - electrons, ions, and neutral atoms & molecules - that make up the plasma. Under some circumstances, the parameters can be different for the different particle species so that, for example, the electron temperature can be much different than the temperatures of ions and neutrals.

In this experiment, the plasma of interest is the "positive column" of a glow discharge maintained in a long tube by a fairly high DC voltage ($\sim$ 1 kV) and small current ($\sim$ a few mA). The tube is placed in series with a fixed ballast resistor that helps select the actual operating current. The discharge has a complex structure, especially near the negative cathode. However, the long uniformly glowing region that extends almost up to the anode is called the positive column and will be seen to have fairly uniform characteristics. In fact, the positive column is *an open non-equilibrium system* sustained by a constant input of energy (due to work done by the electric field on mobile charges) and by a steady flow of current passing through the region. Our goal in this write-up is to create a first model of the postive column plasma in order to make initial estimates of the parameters that we intend to measure.

[Image of plasma]

[Plasma tube circuit]

[Diagram of plasma regions]

[Discharge current-voltage plot]



### Elementary kinetic theory for gases

Before we try to understand plasmas, let's look at ways to describe a macroscopic quantity of gas that consists of a very large number of identical molecules of mass m.

First review the ideal gas law relating pressure $p$, volume $V$, and absolute temperature $T$:

$$pV=NkT$$

Here k is Boltzman's constant = $1.3806\times10^{-23}$ joules/K. Here  pressure is given in pascals ($\text{N/m}^2$), volume in $\text{m}^3$, and absolute temperature in kelvin (K). 

|Question *Q1*: What is the gas number density in number/$\text{cm}^3$ at standard temperature (0°C) and pressure (760torr)?|
|---|

|Question *Q2*: What is the gas number density in number/$\text{cm}^3$ at 20°C and 1 torr? Use the above result multiplied by appropriate ratios of temperatures and pressures.|
|:---|

|Question *Q3*: How many molecules are in a plasma tube of length 80 cm and inner diameter 5 cm at a pressure of 1 torr and temperature  20°C?|
|---|

So far we have regarded temperature as a thermodynamic quantity that specifies the equilibrium condition of the gas: that is, if another body (say a stainless steel electrode) at the same temperature is put into contact with the gas, nothing happens; however, if a body of, say, a higher temperature is introduced, the combined system adjusts to a new equilibrium.

|Question *Q4*: what would happen to the gas temperature and pressure if a warmer electrode were introduced? (Assume the system is closed off so that there is no flow into or out of the tube.)|
|---|

Now we will revise our point of view and think of the gas as a system of many, many molecules in constant motion. Since there are so many molecules, we cannot possibly track the motion of each one. Instead, we can describe the system in statistical terms. To do this, we introduce the concept of a distribution function $f(x,y,z,\text{v}_x,\text{v}_y,\text{v}_z)$. The value $f(x,y,z,\text{v}_x,\text{v}_y,\text{v}_z)dxdydzd\text{v}_xd\text{v}_yd\text{v}_z$ describes how many molecules have positions and velocities contained within the 6-dimensional domain

$$(x,x+dx),(y,y+dy),(z,z+dz),(\text{v}_x,\text{v}_x+d\text{v}_x),(\text{v}_y,\text{v}_y+d\text{v}_y),(\text{v}_z,\text{v}_z+d\text{v}_z).$$

This is a description that makes sense when there is a very large number of molecules in the system, as is the case for macroscopic systems. Moreover, we can deduce macroscopically observable properties from this function.

We hypothesize a separation of the dependence on position from the dependence on velocity. If the gas is in equilibrium with surroundings at temperature T, the distribution function is the product of a (number) density function and the Maxwell-Boltzmann distribution:

$$f(x,y,z,v_x,v_y,v_z)=n(x,y,z)\left(\frac{m}{2\pi kT}\right)^{3/2}e^{-\frac{m({\text{v}_x}^2+{\text{v}_x}^2+{\text{v}_x}^2)}{2kT}}.$$

Note that the temperature can now be interpreted in a new way: the distribution for each velocity component has the form of a Gaussian  whose variance is proportional to the temperature. The fact that this  temperature matches the thermodynamic temperature for a gas in equilibrium is a deep idea connecting statistical mechanics to thermodynamics. We will see below that when we deal with non-equilibrium conditions for a plasma, the statistical interpretation of temperature can still be used even though the thermodynamic interpretation is not strictly valid for a non-equilibrium system. While the Maxwell-Boltzman distribution can be proven to be (by far) the most likely distribution for a gas in equilibrium, it may only be an approximate model for particles in a non-equilibrium plasma.

In terms of molecular speed 

$$\text{v}=\left(\text{v}_x^2+\text{v}_y^2+\text{v}_z^2\right)^{1/2}$$ 

the distribution becomes

$$f_{\text{speed}}(x,y,z,v)=n(x,y,z)\left(\frac{m}{2\pi kT}\right)^{3/2}4\pi \text{v}^2e^{-\frac{m\text{v}^2}{2kT}}.$$

[To be added later: code to generate plot of the speed distribution.]

|Question *Q5*: find - either by derivation or by looking them up - expressions for the following: most probable speed $\text{v}_p$, average speed $\text{v}_\text{av}$, and root-mean-square speed $\text{v}_\text{rms}$. What is the relative ratio $\text{v}_p:\text{v}_\text{av}:\text{v}_\text{rms}$ of these quantities? |
|---|

|Question *Q6*: What is the root-mean-square speed in m/s of a nitrogen molecule at room temperature 20°C? How does this compare to the speed of sound?|
|---|
 
|Question *Q7*: what is the average kinetic energy of a molecule at room temperature 20°C? Give your answer in joules and in electron volts (eV). (Does the mass of the molecule matter?)|
|---|

Note that at room temperature:
  
 $$kT=1.38\times10^{-23}\times293.15=4.05\times10^{-21}\text{J}=0.0253\text{eV}$$
  
  The rule of thumb is that kT is about 1/40 eV at room temperature. This energy scale is much less than typical atomic excitation energies and molecular bond strengths. Thus in order to create a plasma, some means will be needed to provide particles with energy much larger than the average value for a room temperature gas.

### Forming the plasma 

We are going to adopt a point of view that the positive column is fed by a constant influx of energetic electrons drifting down the column under the influence of an electric field created as a result of the applied potential difference between anode and cathode. We assume also that the density of electrons is balanced by a very nearly equal density of positive ions that are drifting much more slowly because of their much larger mass. We way that the plasma is "quasi-neutral", only allowing small differences in the densities to occur under special circumstances that will be explored later.

How this population of electrons and ion comes into being near and just beyond the cathode is a complex process worthy of separate analysis and experimentation. Our goal is to arrive at a model that shows how these populations are maintained in the positive column against population losses due to recombination in the bulk and near the walls and against energy losses as the energetic electrons collide with a much larger population of neutral molecules. Out of this analysis will come predictions of the electron temperature, local electric field strength, and electron/ion densities.

### Sustaining a plasma: it's all about collisions
(Well mostly... we also need to have an electric field that accelerates charged particles between collisions.)

In order to go from gas physics to plasma physics, we need to have a ideas and quantities that describe collision processes. Collisions between neutral molecules are always occuring in normal gases, of course, but in a plasma we rely on collisions as a means to create most of the exciation and ionization of atoms and molecules. (Some initial ionization, enough to start a cascade process, comes from external causes such as cosmic rays and natural radiation.) Another consequence of collisions is to create a distribution of energies for each of the species making up the plasma.

### Collision cross-section and mean free path

Let's first still think only of neutral atoms or molecules. How far must one molecule travel on average before hitting another? We expect the answer to depend on how big each molecule is and how densely the molecules are placed into the containing volume. The size will be defined by a collision cross-section $\sigma$ and density will be given by n.

A simple argument says that the volume swept out by a tube of cross-sectional area $\sigma$ and length equal to the mean-free-path $l_m$ should contain on average one molecule...this is just what is needed to guarantee (on average) that a collision will occur as another molecule travels through this volume.

$$n \sigma l_m = 1$$

So the mean-free-path is given by 

$$l_m = \frac{1}{n \sigma}$$

For identical molecules of diameter d, they will collide if the distance between centers is twice the radius, namely the diameter d itself. A cylindrical volume swept out by a moving molecule before it makes a farthest possible contact with another molecule will have a *radius* equal to this molecular diameter. Thus the collision cross-section can be related to diameter by 

$$\sigma=\pi d^2$$

Mean free path is then related to molecular diameter by:

$$l_m = \frac{1}{n \pi d^2}$$

A table of molecular diameters inferred, in fact, from collision data is found in Wikipedia at:

https://en.wikipedia.org/wiki/Kinetic_diameter

Refined reasoning must account for particles having a distribution of relative velocities, leading to a numerical correction to the above result:

$$l_m = \frac{1}{\sqrt{2} n \pi d^2}$$

However, we will ignore the $\sqrt{2}$ numerical factor for purposes of estimates and in cases where one particle's speed is much higher than the other's speed, such as electrons colliding with neutrals or in beam experiments.

From above, we saw that for an ideal gas (and for practical purposes the gas in this experiment), n = p/kT. Thus

$$l_m = \frac{kT}{p \pi d^2}$$


|Question *Q8*: assuming the gas is entirely made of nitrogen molecules at a temperature of 20°C and pressure of 1 torr (as in Question *Q2* above), what is the mean free path for nitrogen molecules under this condition?|
|---|
  
This sets an important length scale for further consideration of plasma formation and behavior.

|Question *Q9*: suppose we want to make a plasma inside a microchannel or capillary with characteristic dimension of 0.1mm. At what pressure will the mean free path be comparable to this channel size, assuming a temperature of 20°C?|
|---|

In fact, some clever experiments have been done with creating discharge plasmas in networks of micro-channels with width of 0.250 mm and depth of 0.1 mm. See: D. R. Reyes, et al. "Glow discharge in microfluidic chips for visible analog computing,", **Lab on a Chip**, **v.2**, pp. 113-116 (2002). Pressures ranging from roughly 10 to 100 torr were used, so the mean free path was well below the channel dimension.

### Electron-molecule collision cross-section 

But now let's think about electrons colliding with molecules. Since the electrons are much smaller than the molecules, we expect the cross-section to be defined by the radius of a single molecule rather than the sum of radii of the colliding particles. A first estimate of the cross-section will now be

$$\sigma = \pi r^2 = \pi \frac{d^2}{4}.$$

|Question *Q10*: what is the estimated cross-section for an electron colliding with nitrogen?|
|---|
  

### Electron-molecule collision rate and related parameters

We want to develop a way of quantifying the rate at which energetic electrons traveling through a plasma excite, ionize, or elastically scatter from gas molecules. To do this, we want insight into how the collision cross-sections are related to the *rate* at which collisions of a given type occur.

Imagine shooting a BB into a large box whose entrance has area A perpendicular to the path of the BB*. The box contains $n_{\text{target}}$ floating balloons per unit volume, each with cross-sectional area $\sigma.$ If the BB enters at a random location in the cross-section A, it has a chance $\sigma/A$ of hitting any given balloon. If the BB has speed $\text{v}$ then the volume that is probed by the BB in a time interval dt is $dV=A \text{v} dt$ and the average number of targets in this volume is $dN_{\text{target}}=n_{\text{target}} A \text{v} dt$. The average number of hits is the average number of targets multiplied by the probability of hitting any single target:

$$dN_{\text{hit}}=dN_{\text{target}}\frac{\sigma}{A}=n_{\text{target}} A \text{v} dt \frac{\sigma}{A}=n_{\text{target}} \text{v} dt \sigma.$$

The rate of hitting targets as the BB travels through the box is on average

$$\nu \equiv \frac{dN_{\text{hit}}}{dt} = n_{\text{target}} \text{v} \sigma.$$

This would be the average rate with which we hear balloon pops as a BB travels into the box. Alternatively, an average time interval $\tau$ between pops would be given by 

$$\tau=\frac{1}{\nu}=\frac{1}{n_{\text{target}}\text{v}\sigma}.$$

|Question *Q11* A 1 cubic meter box contains 100 mylar balloons with 10 cm diameter. A bb passes through the box at 60 m/s. What is the average collision frequency and average collision time?|
|---|

The mean free path for the BB between hitting successive balloons is given by $l_m=\text{v}\tau$ so we recover our earlier result: 

$$l_m=\frac{1}{n_{\text{target}}\sigma}.$$

Now suppose we arrange for a swarm of BB's to enter the box so that there are $n_{\text{BB}}$ BB's per unit volume traversing the box. Then the collision rate per unit volume – the number of collisions per second per unit volume – is

$$r_{\text{collision}}=n_{\text{BB}}\nu=n_{\text{BB}}n_{\text{target}} \text{v} \sigma=Kn_{\text{BB}}n_{\text{target}}$$

where the rate constant K is given by

$$K \equiv \text{v} \sigma.$$

*NOTE: a wonderful collection of stop-action photos of BB's passing through balloons may be found at:
http://hiviz.com/stock/pages/lmwstock_16.htm


### Estimating the Ionization Rate

The ionization rate due to collision of an electron with neutral gas molecules is found by performing a weighted average of the product $n_g\text{v}\sigma_i$ over the electron speeds using the electron distribution function $f_e(\text{v})$:

$$\nu_i  = \int_{0}^{\infty}n_g\text{v}\sigma_i(\tfrac{1}{2}m_e \text{v}^2,\mathcal{E}_i)f_e(\text{v})d\text{v}$$

Here the ionization cross-section $\sigma_i$ is considered to be a function of the electron kinetic energy $\frac{1}{2}m\text{v}^2$ and gas molecule ionization energy $\mathcal{E}_i$.

For now, we'll assume a Maxwellian distribution parametrized by an "electron temperature" $T_e$. This assumption must ultimately be tested by experiment, leading to possible empirical refinement of the actual distribution function.

$$f_e(\text{v})=\left(\frac{m_e}{2\pi kT_e}\right)^{3/2}4\pi \text{v}^2e^{-\frac{m_e\text{v}^2}{2kT_e}}$$

With this assumption, the average kinetic energy of the electrons is $\left<\frac{1}{2}m_e\text{v}^2\right>=\frac{3}{2}kT_e$.


The postive column is expected to be weakly ionized: only a small fraction of the gas molecules have been converted to ion-electon pairs. This is achievable when only the most energetic electrons have enough energy to cause ionization, so that the average electron kinetic energy is still much less than the ionization energy (perhaps by a factor of 5 or so): $kT_e << \mathcal{E}_i$. Since the tail of the electron velocity distribution is falling off steeply for electron speeds several times larger than the most probable speed, we expect that the ionization is dominated by those electrons whose kinetic energy is close to but slightly greater than the ionization energy $\mathcal{E}_i$. More energetic electrons are simply much less abundant.

With this in mind, we make a linear expansion of the ionization cross-section around the value of the electron kinetic energy where the cross-section just becomes nonzero, i.e. around the ionization energy $\mathcal{E}_i$.

$$\sigma_i \approx C_1\left(\tfrac{1}{2}m\text{v}^2-\mathcal{E}_i\right)$$

when $\frac{1}{2}m\text{v}^2>\mathcal{E}_i$ and 0 otherwise. Here the coefficient $C_1$ is the slope of the linear growth in ionization cross-section as electron kinetic energy exceeds $\mathcal{E}_i$ and is either found empirically from measurements of $\sigma_i$ or from more detailed theoretical models. (e.g. Thomas model...)

Let us define $\sigma_1\equiv C_1\mathcal{E}_i$ so that we can write:

$$\sigma_i \approx \sigma_1\left(\frac{\tfrac{1}{2}m\text{v}^2}{\mathcal{E}_i}-1\right)$$ 

when $\frac{1}{2}m_e\text{v}^2>\mathcal{E}_i$ and 0 otherwise.

We arrive at a calculation of the ionization rate $\nu_i$ that can be done in closed-form to reveal the dependence on electron temperature $T_e$:

$$\nu_i  \approx \int_{\sqrt{2\mathcal{E}_i/m_e}}^{\infty}n_g\text{v}\sigma_1\left(\frac{\tfrac{1}{2}m_e\text{v}^2}{\mathcal{E}_i}-1\right)\left(\frac{m_e}{2\pi kT_e}\right)^{3/2}4\pi \text{v}^2e^{-\frac{m_e\text{v}^2}{2kT_e}}d\text{v}.$$

This integration can be carried out to yield the result:

$$\nu_i \approx n_g\sigma_1\left(\frac{8kT_e}{\pi m_e}\right)^{\frac{1}{2}}\left(1+2\frac{kT_e}{\mathcal{E}_i}\right)e^{-\frac{\mathcal{E}_i}{kT_e}}.$$

|Question *Q12* Derive the above result. One approach is to use integration by parts (several times).|
|---|

Let's rewrite this consistently as a functionof the ratio $\frac{kT_e}{\mathcal{E}_i}$:

$$\nu_i \approx n_g\sigma_1\left(\frac{8\mathcal{E}_i}{\pi m_e}\right)^{\frac{1}{2}}\left(\frac{kT_e}{\mathcal{E}_i}\right)^{\frac{1}{2}}\left(1+2\frac{kT_e}{\mathcal{E}_i}\right)e^{-\frac{\mathcal{E}_i}{kT_e}} \approx n_g\sigma_1\left(\frac{8\mathcal{E}_i}{\pi m_e}\right)^{\frac{1}{2}}\left(\frac{kT_e}{\mathcal{E}_i}\right)^{\frac{1}{2}}e^{-\frac{\mathcal{E}_i}{kT_e}}.$$

Keeping higher-order terms in the expansion of the ionization cross-section would lead to terms with higher powers of $\frac{kT_e}{\mathcal{E}_i}$ in this expression for $\nu_i$.


### Population balance equation for the electron / ion populations

The following equation describes a process of dynamic equilibrium for sustaining the electron/ion populations: diffusion of ions to the walls is balanced against a source term due to ongoing ionization minus a loss term due to bulk recombination.

$$-D_a\left(\frac{d^2}{dr^2}+\frac{1}{r}\frac{d}{dr}\right)n_i=\nu_i n_e-K_rn_in_e$$

Assuming quasi-neutral conditions we take $n_e \approx n_i$ and assuming weak ionization the recombination term $K_rn_in_e$ is neglected so that:

$$D_a\left(\frac{d^2}{dr^2}+\frac{1}{r}\frac{d}{dr}\right)n_i+\nu_i n_i=0.$$

$$\left(\frac{d^2}{dr^2}+\frac{1}{r}\frac{d}{dr}\right)n_i+\frac{\nu_i}{ D_a}n_i=0.$$

The problem is not completely posed until boundary conditions are specified. In this case, we require that the solution remains finite at r=0 (the axis of the discharge tube) and – a key assumption – that ions efficiently combine with electrons at the walls so that $n_i=0$ at $r=R$, where R is the radius of the inside surface of the glass tube containing the electric discharge.

Before specifying the solution, consider an analogous mathematical problem: the calculation of the probability amplitude in Schrödinger's equation for a particle in a one-dimensional box:

$$-\frac{\hbar^2}{2m}\frac{d^2}{dx^2}\psi=E\psi.$$

$$\frac{d^2}{dx^2}\psi+\frac{2mE}{\hbar^2}\psi.$$

Here the boundary conditions are $\psi=0$ at $x=0$ and $x=L$. If we define $k^2\equiv\frac{2mE}{\hbar^2}$,the equation becomes

$$\frac{d^2}{dx^2}\psi+k^2\psi,$$

and the solution is

$$\psi=A\sin kx$$.

The boundary condition at $x=0$ is met by choosing only the sin function and not the cosine function as a solution. The boundary condition at $x=L$ requires that $kL=n\pi$. This in turn leads to quantization of the energy:

$$E_n=\frac{(\hbar k)^2}{2m}=\frac{(n\pi\hbar)^2}{2mL^2}=n^2\frac{h^2}{8mL^2}.$$

Returning to our diffusion problem for the radial diffusion of ions in the positive column, we define $\kappa^2 \equiv \frac{\nu_i}{ D_a}$, so that the equation may be written as:

$$\left(\frac{d^2}{dr^2}+\frac{1}{r}\frac{d}{dr}\right)n_i+\kappa^2n_i=0.$$

The solutions to this kind of equation cannot be trig functions because of the extra $\frac{1}{r}\frac{d}{dr}$ term; instead they are another type of transcendental function called Bessel functions $J_0(\kappa r)$ and $Y_0(\kappa r)$, where the second of these functions goes to infinity at $r=0$ and thus is excluded ... analogous to our exclusion of the cosine solition for the Schrödinger equation.

We must impose the boundary condition that $J_0(\kappa R)=0$. While $\sin(kL)$ has zeros at $kL = \pi,2\pi,3\pi,...=$ 3.1416, 6.2832, 9.4248,... the Bessel function evaluated at $J_0(\kappa R)$ has zeros at $\kappa R=$ 2.4048, 5.5201, 8.6537,... (Note that the Bessel function zeros are not evenly spaced.)

However we can only allow the first zero as a solution because the functions with higher values become negative over part of the domain $r<R$ and we negative ion densities are unphysical.

Thus we require $\kappa R=2.4048$ or $\kappa=\frac{2.4048}{R}$, leading back to a requirement that 

$$\sqrt{\frac{\nu_i}{ D_a}}=\frac{2.4048}{R}.$$

$$\nu_i= D_a\frac{2.4048^2}{R^2}.$$

An expression for the ambipolar diffusion coefficient for non-equilibrium plasmas where $T_e >> T_i \approx T_g$ is (see Raizer, p.29 eq. 2.36):

$$D_a=\mu_i\frac{kT_e}{e}$$

where $\mu_i$ is the ion mobility such that the ion drift velocity in the presence of an electric field is $\text{v}_{\text{d}i}=\mu_iE$. We now have:

$$\nu_i=\mu_i\frac{kT_e}{e}\frac{2.4048^2}{R^2}.$$

This analysis was first described by Schottky in 1924 (see W. Schottky, **Phys. Zeit. v. 25**, 635 (1924)). You might encounter this name again when doing the Hall effect experiment, where we must think about how electrons cross from a metalic contact into a semiconductor material. The concept of a "Schottky barrier" will be introduced. This shows how a physicist can work on many different problems.

### Finding the electron temperature

We can use the above requirement on the ionization rate to find the electron temperature $T_e$ by replacing $\nu_i$ on the left-hand side with its expression in terms of gas density $n_g$, proportionality constant $\sigma_1$ for the ionization cross-section's dependence on energy above the ionization energy $\mathcal{E}_i$, and electron temperature $T_e$:

$$n_g\sigma_1\left(\frac{8\mathcal{E}_i}{\pi m_e}\right)^{\frac{1}{2}}\left(\frac{kT_e}{\mathcal{E}_i}\right)^{\frac{1}{2}}e^{-\frac{\mathcal{E}_i}{kT_e}}=\mu_i\frac{kT_e}{e}\frac{2.4048^2}{R^2}.$$

$$\left(\frac{\mathcal{E}_i}{kT_e}\right)^{-\frac{1}{2}}e^{\frac{\mathcal{E}_i}{kT_e}}=\frac{en_g}{\mu_i}\frac{\sigma_1}{\mathcal{E}_i}\left(\frac{8\mathcal{E}_i}{\pi m_e}\right)^{\frac{1}{2}}\frac{R^2}{2.4048^2}$$

Also recall our definition $\sigma_1\equiv C_1\mathcal{E}_i$ so we can write

$$\left(\frac{\mathcal{E}_i}{kT_e}\right)^{-\frac{1}{2}}e^{\frac{\mathcal{E}_i}{kT_e}}=\frac{e n_g C_1}{\mu_i }\left(\frac{8\mathcal{E}_i}{\pi m_e}\right)^{\frac{1}{2}}\frac{R^2}{2.4048^2}.$$

Define $a \equiv \frac{e n_g C_1}{p_0}$, and $eV_i \equiv \mathcal{E}_i $. Physically $V_i$ is the ionization potential of the gas molecules in volts and $a$ is a measure of the ionization per unit length per unit gas pressure per volt of electron energy above $V_i$. (In this linear relation describing the ionization we would give the electron kinetic energy in volts, obtained by dividing the actual energy $\tfrac{1}{2}m_e\text{v}^2$ in joules by the electron charge e.)

Here we have introduced the gas pressure $p_0$ that would give the number density $n_g$ at temperature $T_0=\text{273.15 K}$. So from the ideal gas law, $p_0 \equiv n_g k T_0$. This means that $a$ is actually a property of the gas: $a = \frac{e C_1}{k T_0}$ and is another measure of the linear approximation of the ionization rate's dependence on the difference between electron energy and ionization energy.

We now have:

$$\left(\frac{eV_i}{kT_e}\right)^{-\frac{1}{2}}e^{\frac{eV_i}{kT_e}}=\frac{a p_0}{\mu_i}\left(\frac{8eV_i}{\pi m_e}\right)^{\frac{1}{2}}\frac{R^2}{2.4048^2}.$$

Multiply top and bottom by the gas pressure $p_0$ and rearrange terms to get:

$$\left(\frac{eV_i}{kT_e}\right)^{-\frac{1}{2}}e^{\frac{eV_i}{kT_e}}=\left(\frac{8e}{\pi m_e}\right)^{\frac{1}{2}}\frac{1}{2.4048^2}\frac{a(V_i)^{\frac{1}{2}}}{\mu_i p_0}(p_0 R)^2.$$

| Question *Q13*: Verify that the units work out if all quantities are in SI units. -------------|
|---|

Define

$$c^2 \equiv \frac{a(V_i)^{\frac{1}{2}}}{\mu_i p_0}$$

Altneratively,

$$c^2=\frac{e n_g C_1(V_i)^{\frac{1}{2}}}{\mu_i p_0}= \frac{e n_g \sigma_1(V_i)^{\frac{1}{2}}}{\mathcal{E}_i\mu_i p_0}$$

We claim that $c^2$ is a property of a given gas that can be computed from measured values of the individual coefficients. A subtle point is that the *product* $\mu_i p_0$ is a property of the gas independent of imposed conditions, since the mobility $\mu_i$ itself is inversely proportional to density and hence to pressure $p_0$. Much of the literature – especially the "classic" references – expresses pressure $p_0$ in torr, radius $R$ in cm, mobility $\mu_i$ in $\frac{\text{cm}^2}{Vs}$,  ionization factor $a$ in ions per (cm x torr x volt), and ionization potential $V_i$ in volts. By carefully working through the conversions we have the numerical relation

$$\left(\frac{eV_i}{kT_e}\right)^{-\frac{1}{2}}e^{\frac{eV_i}{kT_e}}=1.16\times10^7 (c p_0 R)^2.$$

The numerical term has units $\frac{\text{cm}}{\text{s}}\frac{1}{\text{volt}^{\frac{1}{2}}}$, the "Von Engel - Steenbeck constant" $c$ has units $\left(\frac{\text{s}}{\text{cm}}\right)^\frac{1}{2}\frac{\text{volt}^\frac{1}{4}}{\text{torr}\space\text{cm}}$, and the pressure-radius product $p_0 R$ has units (torr cm). Thus the entire combination on the right-hand side of the above equation is dimensionless as is, of course, the left-hand side.

For computational purposes, it is convenient to take the natural logarithm:

$$\frac{eV_i}{kT_e}-\frac{1}{2}\ln\left(\frac{eV_i}{kT_e}\right)=16.27+2\times\ln(c p_0 R).$$

Strictly speaking, this involves taking the log of dimensioned quantities and is numerically valid only for the choices of dimensions described above. One can make a first estimate of the ratio $\frac{eV_i}{kT_e}$ by neglecting the second term on the left-hand side:

$$\frac{eV_i}{kT_e}\approx 16.27+2\times\ln(c p_0 R).$$

The result of this approximate equation can then be used as a first guess using a numerical root finder to refine the solution to the entire equation. (See below for a python code that does this.)

The following is a table of gas constants. For reasons not yet determined, the c values calculated from the given formula only roughly match the published c values. In the end, the temperature estimates are fairly rough anyway, so it does not matter much. It is probably best to used the tabulated value $c_{\text{vEngel}}$ to compare to possible other literature results.

### Ionization data and von Engel - Steenbeck coefficients

$$c^2 \equiv \frac{a(V_i)^{\frac{1}{2}}}{\mu_i p_0}$$

|Ion|$V_i$|$a$|$\mu_i p_0$|$c_\text{calc}$|$c_\text{vEngel}$|$c_\text{Cobine}$|
|------|------|------|------|------|------|------|
|$\text{He}^+$|24.5|0.046|8000|0.0053|0.004|0.0039|
|$\text{Ne}^+$|21.5|0.056|3300|0.0089|0.006|0.0059|
|$\text{Ar}^+$|15.7|0.71|1200|0.048|0.04|0.053|
| | | | | | | |
|$\text{Hg}^+$|10.4|0.83|230|0.11|0.07|0.11|
| | | | | | | |
|$\text{H}_2^+$|15.4|0.21|10000|0.0091|0.01|0.0135|
|$\text{N}_2^+$|15.5|0.26|2000|0.023|0.04|0.035|
|$\text{O}_2^+$|12.1|0.24|1000|0.029|     |     |
| | | | | | | |
|vEngel|p59T3.6|p63T3.7|p114T4.1|     |p243F124|  |
|Cobine|p80T4.1|p83T4.4|     |     |    |p.240T8.8|

https://en.wikipedia.org/wiki/Ionization_energies_of_the_elements_(data_page)

### Python code to estimate the electron temperature
The following code will estimate the temperature for a given gas after you edit the values for pressure p, tube radius R, ionization potential Vi, and von Engel - Steenbeck constant c. 

In [6]:
# 7 January 2017 Randall Tagg (revised 8 Jan 17)
# This computes a predicted electron gas temperature for given
# tube pressure, tube radius, and published gas constants.
# The function scipy.optimize.fsolve is used to find the zero of 
# transcendental expression derived by von Engel and Steenbeck
# for the ratio of gas ionization potential energy to electron 
# temperature.
#
import math
import scipy.optimize as opt
#
p = 0.592 #tube pressure in torr
R = 2.54 #tube radius in cm
#
# For choices of gas constants see a table in the jupyter document
# in which this embedded or else see p. 59 Table 4.3 and p. 197 of
# Y. P. Raizer, "Gas Discharge Physics (Springer Verlag, 1991)
#
# The following gas constants are for nitrogen
Vi = 15.5 #ionization potential in volts
c = 0.04 #von Engel-Steenbeck consant (volts)^0.25(cm/s)^-0.5(torr-cm)^-1
#
# The following gas constants are for neon
###Vi = 21.5 #ionization potential in volts
###c = 0.006 #von Engel-Steenbeck consant (volts)^0.25(cm/s)^-0.5(torr-cm)^-1
#
cpRterm = 16.27+2*math.log(c*p*R)
#etempRatioGuess = Vi/etempGuess
etempRatioGuess = cpRterm
print('{0:.3f} eV {1:.0f} K - first guess of electron temperature'.format(Vi/etempRatioGuess,11605*Vi/etempRatioGuess)) 
#
# Here is the von Engel - Steenbeck transcendental function
def etempfunc(etempRatio,cpRterm):
#    fvalue = math.exp(etempRatio)-numfac*math.sqrt(etempRatio)
    fvalue = etempRatio-0.5*math.log(etempRatio)-cpRterm
#    print(fvalue)
    return fvalue
#
# Find and print the zero of the above function
# See: https://docs.scipy.org/doc/scipy-0.14.0/reference/generated/scipy.optimize.fsolve.html
etempRatio=opt.fsolve(etempfunc,etempRatioGuess,args=(cpRterm))
etemp=Vi/etempRatio
print('{0:.3f} eV {1:.0f} K - refined estimate of electron temperature'.format(etemp[0],11605*etemp[0])) 
#
#The following legacy code tested the root solver fsolve
#def quad(x):
#    return x*x-2*x+1
#print(opt.fsolve(quad,0.7))

1.456 eV 16893 K - first guess of electron temperature
1.304 eV 15134 K - refined estimate of electron temperature


| Question *Q14*: (This requires having your own version of the  document running within Jupyter so that you can actually edit and execute code.) Using the above code with the parameters Vi and c chosen for nitrogen, edit the value of pressure p over the range 0.5 to 1.5 torr in steps of 0.1 torr. Run the code for each cse to obtain the refined estimate of electron temperature. Then tabulate the results and plot the electron temperature versus pressure. You can do this manually one edit at a time or you can edit the entire code to loop through the values and make a plot. If doing the latter, copy the code into a new cell and edit the copied version.|
|---|



### References

Raizer, y. P., *Gas Discharge Physics* (Springer, 1991).
(Several other books have been consulted, too, and will be added to this list. However, Raizer is the primary reference.)